src/frontend/sharedComponents/BottomSheet/index.tsx

```diff
@@ -1,21 +1,32 @@
 import * as React from "react";
-import { BackHandler } from "react-native";
+import { BackHandler, View } from "react-native";
 import {
   BottomSheetModal,
   BottomSheetModalProvider,
 } from "@gorhom/bottom-sheet";
 
-import Backdrop from "./Backdrop";
+import { Backdrop } from "./Backdrop";
 
-const SNAP_POINTS = [400];
+const DEFAULT_SNAP_POINTS = [400];
 
 interface Props extends React.PropsWithChildren<{}> {
+  hideDragHandle?: boolean;
   onHardwareBackPress?: () => void;
   onDismiss: () => void;
+  snapPoints?: (string | number)[];
 }
 
-const BottomSheet = React.forwardRef<BottomSheetModal, Props>(
-  ({ children, onHardwareBackPress, onDismiss }, ref) => {
+export const BottomSheet = React.forwardRef<BottomSheetModal, Props>(
+  (
+    {
+      children,
+      hideDragHandle = false,
```

Comment:
depending on design feedback we may want to change the default to hiding the handle and renaming the prop to something like `showDragHandle`